zebra/interface.c

```diff
@@ -1664,19 +1778,33 @@ static void interface_update_stats(void)
 #endif /* HAVE_NET_RT_IFLIST */
 }
 
+static int parse_ipa_type(const char *ip_version)
```

Comment:
Return should be of type enum ipaddr_type_t;

Reply:
Fixed it.